priv/base/runner

```diff
@@ -222,6 +222,9 @@ do_start() {
 }
 
 do_stop() {
+    # Note: We don't ship with a -shutdown_time vm argument in release
+    #       packages, so there's no hard ceiling.
+    echo "Gracefully stopping may take a few minutes."
```

Comment:
An improvement may be to recommend tailing console.log to check progress if shutdown is taking a long time.